packages/cli/src/commands/build.ts

```diff
@@ -710,7 +710,9 @@ function expandBuild(files: string[], build: Builder): Builder[] {
     });
   }
 
-  let src = normalize(build.src || '**');
+  let src = normalize(build.src || '**')
```

Comment:
Curious, would `posix.normalize()` not work here?

Reply:
I tried that and I believe `normalize()` simply resolves `.` and `..` in the path, it doesn't change the path separators:

```
> path.posix.normalize('C:\\foo\\bar\\baz')
'C:\\foo\\bar\\baz'
```

Reply:
Fixed!